pkgs/applications/science/astronomy/kstars/default.nix

```
@@ -1,30 +1,30 @@
 {
-  stdenv, extra-cmake-modules, fetchurl,
+  stdenv, mkDerivation, extra-cmake-modules, fetchurl,
 
   kconfig, kdoctools, kguiaddons, ki18n, kinit, kiconthemes, kio,
-  knewstuff, kplotting, kwidgetsaddons, kxmlgui, wrapQtAppsHook,
+  knewstuff, kplotting, kwidgetsaddons, kxmlgui,
 
   qtx11extras, qtwebsockets,
 
   eigen, zlib,
 
-  cfitsio, indilib, xplanet, libnova, gsl
+  cfitsio, indilib, xplanet, libnova, gsl, stellarsolver
 }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "kstars";
-  version = "3.4.3";
+  version = "3.5.0";
 
   src = fetchurl {
-    url = "https://mirrors.mit.edu/kde/stable/kstars/kstars-${version}.tar.xz";
-    sha256 = "0j5yxg6ay6sic194skz6vjzg6yvrpb3gvypvs0frjrcjbsl1j4f8";
+    url = "https://mirrors.ocf.berkeley.edu/kde/stable/kstars/kstars-${version}.tar.xz";
```

```suggestion
    url = "mirror//kde/stable/kstars/kstars-${version}.tar.xz";
```

This should automatically choose an appropriate mirror.

---

Awesome, (assuming you meant mirror://) wasnt aware of that capability. Its now correctly pointing at mirror://kde/stable/kstars/kstars-${version}.tar.xz